core/src/main/java/org/mule/runtime/core/internal/exception/TemplateOnErrorHandler.java

```diff
@@ -263,4 +279,10 @@ public void setHandleException(boolean handleException) {
   public void setErrorType(String errorType) {
     this.errorType = errorType;
   }
+
+  public void setRootContainerName(String rootContainerName) {
```

Comment:
Think it's seen this code (or very similar) in multiple places, ideally put the body in a static helper and reuse.